components/idenstatereader/idenstatereader.go

```diff
@@ -62,7 +66,7 @@ func (s *IdenStateRead) GetState(id *core.ID) (*proof.IdenStateData, error) {
 }
 
 // GetState returns the Identity State of the given ID closest to the blockN
```

Comment:
I thought that it was the closest and **smaller** to the blockN? not sure...

Reply:
yes, closest under (or equal) to the current time (the current valid one (last one) on the current blockN/timestamp)